spatialscaper/utils.py

```diff
@@ -507,7 +507,7 @@ def IR_normalizer(IRs):
         impulse_responses = np.array([[0.1, 0.2, 0.3], [0.4, 0.5, 0.6]])
         normalized_IRs = IR_normalizer(impulse_responses)
     """
-    E = np.sqrt(np.sum(np.power(IRs, 2), axis=-1, keepdims=True))
+    E = np.sqrt(np.sum(np.power(np.abs(IRs), 2), axis=-1, keepdims=True))
```

Comment:
I believe `np.abs` is redundant here.

Reply:
I added it just in case this function is ever used with a complex-valued number, such as the fft of a real-valued signal.